server/app.py

```diff
@@ -16,8 +16,17 @@
 migrate = Migrate(app, db)
 bcrypt = Bcrypt(app)
 jwt = JWTManager(app)
+
+# Stripe setup
 stripe.api_key = Config.STRIPE_SECRET_KEY
 
+""" stripe.WebhookEndpoint.create(
+    url="http://a42d3d4cffbc.ngrok.io/api/v1/payment/payment-method",
```

Comment:
perhaps just store an environment variable for this incase we forget to comment it out every time :)

Reply:
Great idea, will do!